Makefile

```diff
@@ -34,7 +34,7 @@ endif
 
 test-coverage:
 	rm -rf coverage .coverage
-	$(PYTEST) sklearn --show-locals -v --with-cov sklearn
+	$(PYTEST) sklearn --showlocals -v --cov=sklearn --cov-report=html:coverage
```

Comment:
Sorry for this, it slips through the cracks it seems ... I checked locally that it was generating the html coverage fine as was the nose equivalent command.